addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/rain.xml

```diff
@@ -21,6 +21,26 @@
 			<channel id="batteryLevel" typeId="system.battery-level" />
 			<channel id="lowBattery" typeId="system.low-battery" />
 		</channels>
+		
+		<config-description>
+            <parameter name="deviceId" type="text" required="true">
```

Comment:
Can you pls apply the code formatter? You have a mix of spaces and tabs in all XML files now.